modules/dns/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "fqdn" {
+  value = trimsuffix("${var.hostname}.${data.google_dns_managed_zone.dnszone.dns_name}", ".")
```

Comment:
I don't believe this trimsuffix() is needed, at least not in all cases.  In my case, even though the DNS name in the GCP console was 'corrigan.hashidemos.io.', the value that was getting populated in the Admin UI after trimming was 'corrigan.hashidemos.i' (it was trimming the 'o').  Whenever I removed the trimsuffix() and set the output value to:

value = "${var.hostname}.${data.google_dns_managed_zone.dnszone.dns_name}"

It worked as expected and the trailing '.' was not present.
It was trimming the o

Reply:
The trim makes the value more common through the usage. I've fixed the overtrim as well.